modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/EditKBArticleDisplayContext.java

```diff
@@ -107,6 +111,21 @@ public String getContent() {
 			BeanPropertiesUtil.getString(_getKBTemplate(), "content"));
 	}
 
+	public String getDatePickerFormattedDisplayDate() {
+		KBArticle kbArticle = getKBArticle();
+
+		if (kbArticle == null) {
+			return StringPool.BLANK;
+		}
+
+		Format displayFormatDate =
```

Comment:
Please rename variable to `dateFormat`, which is what the `getSimpleDateFormat` is returning. Other valid names IMO would be `format` or `simpleDateFormat`.